tutorons/python/tags.py

```diff
@@ -0,0 +1,26 @@
+#! /usr/bin/env python
```

Comment:
This file might not be necessary for your commit---you may be able to delete it altogether for the Python Tutoron.